packages/slate-editor/src/modules/editor/Editor.tsx

```diff
@@ -582,7 +585,6 @@ export const Editor = forwardRef<EditorRef, EditorProps>((props, forwardedRef) =
                         availableWidth={availableWidth}
                         containerRef={containerRef}
                         events={events}
-                        newsroomSettingsUrl={withPressContacts.newsroomSettingsUrl}
```

Comment:
This prop wasn't really used here. It has been received from the external code and then passed as is to `renderSearch()`, which is external code too. The scheme can be simplified by letting the external code solve it without proxying the prop through the editor.